From 395f4bf89616e096fc6fd95d7baed7dd3219525b Mon Sep 17 00:00:00 2001
From: "Min Hu (Connor)" <humin29@huawei.com>
Date: Tue, 20 Apr 2021 16:59:50 +0800
Subject: [PATCH 131/189] net/hns3: ignore devargs parsing return

In hns3 PMD, as the handler always return 0, the return value
of a function 'rte_kvargs_process' no need to be checked. But
the API definition has return value, so 'void' could be used
to ignore that.

Fixes: a124f9e9591b ("net/hns3: add runtime config to select IO burst function")

Signed-off-by: Min Hu (Connor) <humin29@huawei.com>
---
 drivers/net/hns3/hns3_ethdev.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/net/hns3/hns3_ethdev.c b/drivers/net/hns3/hns3_ethdev.c
index e2a96c1..81c67e2 100644
--- a/drivers/net/hns3/hns3_ethdev.c
+++ b/drivers/net/hns3/hns3_ethdev.c
@@ -7268,11 +7268,11 @@ hns3_parse_devargs(struct rte_eth_dev *dev)
 	if (!kvlist)
 		return;
 
-	rte_kvargs_process(kvlist, HNS3_DEVARG_RX_FUNC_HINT,
+	(void)rte_kvargs_process(kvlist, HNS3_DEVARG_RX_FUNC_HINT,
 			   &hns3_parse_io_hint_func, &rx_func_hint);
-	rte_kvargs_process(kvlist, HNS3_DEVARG_TX_FUNC_HINT,
+	(void)rte_kvargs_process(kvlist, HNS3_DEVARG_TX_FUNC_HINT,
 			   &hns3_parse_io_hint_func, &tx_func_hint);
-	rte_kvargs_process(kvlist, HNS3_DEVARG_DEV_CAPS_MASK,
+	(void)rte_kvargs_process(kvlist, HNS3_DEVARG_DEV_CAPS_MASK,
 			   &hns3_parse_dev_caps_mask, &dev_caps_mask);
 	rte_kvargs_free(kvlist);
 
-- 
2.7.4

